docs/bread/relationships.md

```diff
@@ -35,6 +35,22 @@ If you need to set advanced options for `belongsToMany` relationship you can set
 }
 ```
 
+## Sorting relationships
+
+You can sort the results of a relationship by setting the `sort` object in the relationship options:
+
+```php
+{
+    "sort": {
+        "field": "my_field",
+        "direction": "asc"
+    }
+}
+```
+
+This will sort the results ascending by the field `my_field`. Use `desc` as the direction to sort it in the opposite order.
```

Comment:
If setting `desc` in the example this can be changed as:
```
This will sort the results descending by the field `my_field`. Default value for direction is `asc`.
```